lib/brains/views/webhook.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from lib.brains.client import get_client
+from lib.brains.forms import WebhookParseForm, WebhookVerifyForm
+from solitude.errors import FormError
+from solitude.logger import getLogger
+
+log = getLogger('s.brains')
+
+
+@api_view(['POST'])
+def parse(request):
+    form = WebhookParseForm(request.DATA)
+    if not form.is_valid():
+        raise FormError(form.errors)
+
+    parsed = get_client().WebhookNotification.parse(*form.braintree_data)
```

Comment:
I'm not a fan of putting the braintree call data into the form because it seems like an unnecessary abstraction. You can't glance at this call and see what data is passed in, you have to switch to the form code. It's fine to keep it as is because it's consistent with the rest of solitude, just thought I'd mention it.